src/ContaoCommunityAlliance/DcGeneral/Contao/Dca/Populator/BackendViewPopulator.php

```diff
@@ -159,6 +160,11 @@ protected function populatePanel(EnvironmentInterface $environment)
                 }
             }
         }
+
+        $baseConfig    = $environment->getBaseConfigRegistry()->getBaseConfig();
+        $listingConfig = $viewDefinition->getListingConfig();
+
+        ViewHelpers::initializeSorting($panel, $baseConfig, $listingConfig);
```

Comment:
I just realized that my approach won't work at all because the BaseConfigRegistry just return a clone of the base config. This is imho not the way a registry acts so I did not expect it.